pkgs/applications/graphics/xfig/default.nix

```diff
@@ -35,6 +36,7 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     sed -i 's:"fig2dev":"${fig2dev}/bin/fig2dev":' src/main.c
+    sed -i 's:/usr/bin/:'$out'/bin/:' xfig.desktop
```

Comment:
Normally we use subsituteInPlace for this kind of stuff because it logs when no match is found.

Reply:
I'll look into that.